python-backend/tests/search/resource/test_search_resource.py

```diff
@@ -0,0 +1,84 @@
+import json
+import uuid
+import pytest
+
+from tests.factories import MineFactory, MinePartyAppointmentFactory, PartyFactory
+
+
+# GET
+def test_get_no_search_results(test_client, db_session, auth_headers):
+    get_resp = test_client.get(
+        f'/search?search_term=Abbo', headers=auth_headers['full_auth_header'])
+    get_data = json.loads(get_resp.data.decode())
+    assert get_resp.status_code == 200
+    assert get_data['search_terms'] == ['Abbo']
+    assert len(
+        [key for key, value in get_data['search_results'].items() if len(value) is not 0]) == 0
+
+
+def test_search_party(test_client, db_session, auth_headers):
+    party = PartyFactory(person=True)
+    get_resp = test_client.get(
+        f'/search?search_term={str(party.first_name)}', headers=auth_headers['full_auth_header'])
+    get_data = json.loads(get_resp.data.decode())
+    parties = get_data['search_results']['party']
+    assert len(parties) == 1
+    assert party.first_name in parties[0]['result']['name']
+    assert uuid.UUID(parties[0]['result']['party_guid']) == party.party_guid
+    assert len([
+        key for key, value in get_data['search_results'].items()
+        if key is not 'party' and len(value) is 0
+    ]) == 4
```

Comment:
why 4? should this be dynamic? maybe a comment?

Reply:
@JustinMacaulay thoughts?